go/vt/sqlparser/sql.y

```diff
@@ -4541,6 +4557,18 @@ alter_table_statement_part:
   {
     $$ = &DDL{Action: AlterStr, AlterCollationSpec: &AlterCollationSpec{CharacterSet: "", Collation: $4}}
   }
+| SECONDARY_ENGINE equal_opt ID
+  {
+    $$ = &DDL{Action: AlterStr}
+  }
+| SECONDARY_ENGINE equal_opt STRING
+  {
+    $$ = &DDL{Action: AlterStr}
+  }
+| SECONDARY_ENGINE_ATTRIBUTE equal_opt STRING
```

Comment:
only STRING?

Reply:
Weirdly, yeah. `SECONDARY_ENGINE` can do both, but not `SECONDARY_ENGINE_ATTRIBUTE`. They seem to be different table options.